lib/waterline/query/finders/basic.js

```diff
@@ -395,6 +404,15 @@ module.exports = {
     });
   },
 
+  beforeCallbacks:function(criteria,cb){
```

Comment:
It's not clear to me why this function exists. All it does is call `callbacks.beforeFind`, but it's wrapped in this `beforeCallbacks` function that then wraps that one call in `async.series`. Is the idea that eventually there may be more than just `beforeFind`?